modules/ting_das/js/ting_das.js

```diff
@@ -0,0 +1,22 @@
+(function ($) {
+  "use strict";
+
+  // The click event on the button is blocked by Drupals ajax mechanism so we use ajaxSend to detect
+  // that a ajax call is happening and check if comes from pressing the order button
```

Comment:
Have you tried attaching an event?

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

> addEventListener() works by adding a function or an object that implements EventListener to **the list of event listeners** for the specified event type on the EventTarget on which it's called.

It should be possible to have more than one EventListener on a single element.